tutorials/scripting/gdextension/gdextension_cpp_example.rst

```diff
@@ -388,6 +389,8 @@ loaded for each platform and the entry function for the module. It is called ``g
 This file contains a ``configuration`` section that controls the entry function of the module.
 You should also set the minimum compatible Godot version with ``compatability_minimum``,
 which prevents older version of Godot from trying to load your extension.
+The ``reloadable`` flag enables your extension to be automatically loaded by the editor every time you recompile it,
```

Comment:
```suggestion
The ``reloadable`` flag enables your extension to be automatically reloaded by the editor every time you recompile it,
```

Reply:
Is there an easy way to squash these commits on github without needing to clone the repo and do it locally?

Reply:
I don't think so to be honest. The only thing is to have GitHub do it with Merge and Squash but that's not allowed here due to the commit messages being just merged together which isn't clean. You will need to do that locally then